packages/app/package.json

```diff
@@ -20,8 +20,9 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@cypress-design/vue-icon": "0.18.1",
-    "@cypress-design/vue-statusicon": "0.2.4",
+    "@cypress-design/css": "0.11.0",
```

Comment:
Why does `@cypress-design/css` need to be here as a dependency?  It is used via frontend-shared

Reply:
nah, I think you are right It was a peace of mind step for me.
I will clean up.